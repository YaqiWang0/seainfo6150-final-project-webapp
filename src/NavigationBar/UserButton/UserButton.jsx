import React, { useEffect, useState} from "react"
import  style from "./UserButton.module.css"

const UserButton = (props) => {
    return (

            <nav>
                <ul>
                    <li onClick={props.onClick}>
                        {props.text}
                        <span></span><span></span><span></span><span></span>
                    </li>
                </ul>
            </nav>
    )
}

export default UserButton