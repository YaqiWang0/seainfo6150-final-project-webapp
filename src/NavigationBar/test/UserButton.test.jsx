
import React from 'react'
import UserButton from "../UserButton/UserButton";


describe("UserButton test", () => {

    it('render successfully', function () {
        const {container} = render(
            <UserButton text = {"login"}/>
        );
        expect(container).toMatchSnapshot();
    })
})