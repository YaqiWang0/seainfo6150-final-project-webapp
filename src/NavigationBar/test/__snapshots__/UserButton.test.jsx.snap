// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBar test render successfully 1`] = `
<div>
  <nav>
    <ul>
      <li>
        login
        <span />
        <span />
        <span />
        <span />
      </li>
    </ul>
  </nav>
</div>
`;

exports[`UserButton test render successfully 1`] = `
<div>
  <nav>
    <ul>
      <li>
        login
        <span />
        <span />
        <span />
        <span />
      </li>
    </ul>
  </nav>
</div>
`;
